cfgov/v1/util/date_filter.py

```diff
@@ -2,20 +2,25 @@
 
 from dateutil.relativedelta import relativedelta
 
+# This utility file exists to support date input fields that are typeable
+# text inputs, which occur in browsers that don't support the date input field
+# https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date
 
 # For more information on date formatting, see the python documentation here:
 # https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior
 
 full_date_patterns = (
     '%m/%d/%Y',     # 10/25/2016, 9/1/2016
     '%m-%d-%Y',     # 10-25-2016, 9-1-2016
+    '%Y-%m-%d',     # 2016-10-25, 2016-9-1
```

Comment:
`%m` and `%d` are zero-padded so it'd be cool if you could update all these comments accordingly (`2016-09-01`, `09-01-2016`, etc.) See https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior.

Reply:
Good catch!